doc/examples/datasets_train/datasets_train.py

```diff
@@ -517,16 +647,12 @@ def train_func(config):
     use_gpu = args.use_gpu
     use_s3 = args.use_s3
     dir_path = args.dir_path
-    large_dataset = args.large_dataset
-
-    if large_dataset:
-        assert use_s3, "--large-dataset requires --use-s3 to be set."
 
     start_time = time.time()
 
     ray.init(address=address)
 
-    make_and_upload_dataset(dir_path)
+    make_and_upload_dataset(dir_path, upload_to_s3=use_s3)
```

Comment:
Added some logic to also write data to S3, but again this won't work for the user unless they change the bucket name themselves